app/src/main/res/layout/content_about_mac.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:tools="http://schemas.android.com/tools"
+android:id="@+id/content_about_mac"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+tools:context="in.ac.bits_pilani.goa.ard.activities.AboutMAC"
+tools:showIn="@layout/activity_about_mac">
+
+    <ImageView
+        android:id="@+id/about_mac_image"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/about_mac_image_height"
+        android:scaleType="centerCrop"
+        android:background="@color/blue_900"
```

Comment:
i have removed the background color as it is centercrop hence no background role.